products/vpcaccess/terraform.yaml

```
@@ -24,9 +24,7 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         name: "vpc_access_connector"
         primary_resource_id: "connector"
         vars:
-          # The VPC connector has a name limit of 25 characters
-          # and we add "tf-test" (len 7) prefix and random suffix (len = 10)
-          name: "vpcconn"
+          name: "tf-test-vpcon"
```

this adds a double prefix because of https://github.com/GoogleCloudPlatform/magic-modules/blob/master/provider/terraform/examples.rb#L192
```suggestion
          name: "vpc-con"
```

---

Oh, right, I must have found the wrong place this was happening. I found hundreds of abandoned VPC connections in the CI project with the name `vpcconn${random}`

---

Any idea why this was the previous name? https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-4367-old..auto-pr-4367#diff-9ff4c4e3ca370922061d9bb3cbde019f39c865ec1150c76e26cc945cd34cdb17L57

---

Oh it's in your link, it checks for the presence of `-`

---

yeah maybe there should be a different indicator of whether or not to prefix, but I guess it kinda gets the job done